scripts/scripts_test_utils.py

```diff
@@ -0,0 +1,238 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common utilities for test classes."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import signal
+
+import python_utils
+
+import psutil
+
+
+class PopenStub(python_utils.OBJECT):
+    """Stubs the API of psutil.Popen() to make unit tests less expensive.
```

Comment:
Give more detail here about the savings, rationale, etc.

Reply:
Done.